wolfssl/wolfcrypt/settings.h

```diff
@@ -409,6 +412,16 @@
     #endif
 #endif
 
+#ifdef WOLFSSL_EMBOS
+    #define USE_FAST_MATH
```

Comment:
If the port is truly only settings then these should probably be example emboss settings.  What if someone doesn't want ecc?